src/pages/en/core-concepts/routing.md

```diff
@@ -94,6 +94,14 @@ In this example, a request for `/withastro/astro/tree/main/docs/public/favicon.s
 }
 ```
 
+## Excluding pages
```

Comment:
```suggestion

## Excluding pages

You can exclude pages, or even whole directories from being built by prefixing their names with an underscore (`_`).

This allows you to co-locate tests, utilities, and components with their related pages and prevent them from being built into html files and placed into the `dist/` directory.

```
└── pages/
   ├── posts/
   │   ├── post1.md
   │   ├── SomeComponent.astro
   │   └── utils.js
   ├── _hidden-directory/
   |   ├── page1.md
   |   └── page2.md
   ├── _hidden-page.astro
   └── index.astro
   
```

In the above example, only `src/pages/index.astro` and `src/pages/post1.md` will be built as page routes and html files.
```
What do you think about an example like this, that shows the kinds of files you were talking about? (Feel free to change it to make it more realistic.) Just a thought.

Since this is the page on routing, I did want to slip in the phrase "page route" somewhere in this description, too.

Reply:
Ah Im sorry buddy, I didnt see your comment, I posted my response below,
It's funny how we both had the same idea... 😂

Reply:
Alright, I added a slightly modified version of this to the section under an example heading.